### ISSUE TITLE
Insult.py Help!

### ISSUE THREAD

**Issue by Lumber-Jones** [Lumber-Jones/True-Grit#2]
We can't seem to figure out why the insult generator isn't working in the bot... Any thoughts?

**Comment by corscheid** [comment]
Yeah. My thought is... where is `insults.yml`? Forgot to commit that? or file doesn't exist? I would imagine that it's a text file (yaml format) full of insult texts.

**Comment by Lumber-Jones** [comment]
So a fix has been established in V1.2 that just puts all of the insults into 3 separate arrays and then randomly generates them into a string that gets logged 

`if (command === "insult"){
	
var insultLine1 = ["Artless", "Bawdy", "Beslubbering", "Bootless", "chrlish", "Cockered", "clouted", "craven", "currish", "dankish", "dissembling", "droning", "Errant", "fawning", "fobbing", "froward", "frothy", "Gleeking", "Goatish", "Gorbellied", "Impertinent", "Infectious", "Jarring", "Loggerheaded", "lumpish", "Mammering", "Mangled", "Mewling", "paunchy", "Pribbling", "Puking", "Puny", "Qualling", "Rank", "Reeky", "Rougish", "Ruttish", "Saucy", "Spleeny", "Spongy", "Surly", "Tottering", "Unmuzzled", "Vain", "Venomed", "Villanious", "Warped", "Wayward", "Weedy", "Yeasty"];
	
	var insultLine2 = ["Base-court", "Bat-fowling", "Beef-witted", "Beetle-headed", "Boil-brained", "Clapper-clawed", "Clay-brained", "Common-kissing", "Crook-pated", "Dismal-dreaming", "Dizzy-eyed", "doghearted", "Dread-bolted", "Earth-vexing", "Elf-skinned", "Fat-kidneyed", "Fen-sucked", "Flap-mouthed", "Fly-bitten", "Folly-fallen", "Fool-born", "Full-gorged", "Guts-griping", "Half-facced", "hasty-witted", "Hedge-born", "hell-hated", "Idle-headed", "Ill-breeding", "Ill-nurtured", "Knotty-pated", "Milk-livered", "Motley-minded", "Onion-eyed", "Plume-plucked", "Pittle-deep", "Pox-marked", "Reeling-ripe", "Rough-hewn", "Rude-growing", "Rump-fed", "Shard-borne", "Sheep-biting", "Spur-galled", "Swag-bellied", "tardy-gaited", "Tickle-brained", "Toad-spotted", "Unchin-snouted", "Weather-bitten"];
	
	var insultLine3 = ["Apple-john", "baggage", "barnacle", "bladder", "boar-pig", "bugbear", "bum-bailey", "canker-blossom", "Clack-dish", "Clotpole", "coxcomb", "codpiece", "Death-token", "Dewberry", "Flap-dragon", "Flax-wench", "Flirt-gill", "Foot-licker", "fustilarian", "giglet", "Gudgeon", "Haggard", "Harpy", "Hedge-pig", "Horn-beast", "Hugger-mugger", "Joithead", "Lewdster", "Lout", "Maggot-pie", "Malt-worm", "Mammet", "Measle", "Minnow", "Miscreant", "Moldwarp", "Mumble-news", "Nut-hook", "Pigeon-egg", "Pignut", "Puttock", "Pumpion", "Ratsbane", "Scut", "Skainsmate", "Strumpet", "Varlot", "Vassal", "Whey-face", "Wagtail"];
	
	var InsultRandom = Math.floor(Math.random()* 49);
	
	message.channel.send(`${message.mentions.users.first()} Thou art a  ${insultLine1[InsultRandom]} ${insultLine2[InsultRandom]} ${insultLine3[InsultRandom]}`);
}`

**Comment by Billy-Davies-2** [comment]
I've got some ideas, I'll do a pr when I get them working

**Comment by Billy-Davies-2** [comment]
Please see #4 and  let me know what you think.
